examples/cfg/cfg.json

```diff
@@ -8,6 +8,8 @@
                     "openstack_user": "admin",
                     "openstack_pass": "admin",
                     "openstack_tenant": "admin",
+                    "openstack_domain_id": "",
```

Comment:
When using API v3 you need to specify one of the `openstack_domain_id` or `openstack_domain_name`. If the user specifies both of them they get an error that one can specify only one of them.
However both of them need to specified in global config as due to usage of function `config.GetConfigItem()` all the fields in config are mandatory. 

It may be a bit confusing for the user. So:
* we should add note to documentation that all the config options are mandatory, and one of options `openstack_domain_id` or `openstack_domain_name` needs to be set to an empty string when using v3.
* open an issue to refactor getting config items